lib/spans/batch-span-streamer.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const logger = require('../logger').child({ component: 'batch-span-streamer' })
+const BaseSpanStreamer = require('./base-span-streamer')
+
+class BatchSpanStreamer extends BaseSpanStreamer {
+  constructor(licenseKey, connection, metrics, queueSize) {
+    super(licenseKey, connection, metrics, queueSize)
+    this.sendTimer = null
+    this.batchSize = 750
```

Comment:
It would be nice to have a reminder in a comment here as to how we got to this 750 value.

Reply:
actually I explain value in sendQueue but will update to be a little more clear